http/rest/handler.go

```diff
@@ -0,0 +1,107 @@
+package rest
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"raccoon/config"
+	"raccoon/logger"
+	"raccoon/metrics"
+	"raccoon/pkg/collection"
+	"raccoon/pkg/deserialization"
+	"raccoon/pkg/identification"
+	pb "raccoon/pkg/proto"
+	"raccoon/pkg/serialization"
+	"time"
+)
+
+type Handler struct{}
+
+func (h *Handler) GetRESTAPIHandler(c collection.Collector) http.HandlerFunc {
+	return func(rw http.ResponseWriter, r *http.Request) {
+		contentType := r.Header.Get("Content-Type")
```

Comment:
I wanted to choose JSON when the content-type header is missing but you are right if some other content-type is present apart from JSON/proto it will choose JSON and serialization will fail. Let's fail the request only.